shims/pom.xml

```diff
@@ -32,22 +32,82 @@
     <version>21.10.0-SNAPSHOT</version>
 
     <profiles>
+        <profile>
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <modules>
+                <module>spark301emr</module>
+                <module>spark301</module>
+                <module>spark302</module>
+                <module>spark303</module>
+                <module>spark311</module>
+                <module>spark312</module>
+                <module>spark311cdh</module>
+                <module>aggregator</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>release301</id>
+            <modules>
+                <module>spark301</module>
+                <module>aggregator</module>
```

Comment:
Can this be moved out to a top-level `modules` regardless of profile?  Seems we would always want to traverse that child module.